src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
```

Comment:
We can use illuminate/cache if you think it is better. I've thought about the default cache driver to be SQLite3 or other than file, but they are not supported on every system. So I started with filesystem cache, which is supported. Of course multiple cache drivers support is on the way since if we have more than 10000 users having conversation than more than 10000 cache files are created, which is definetely a problem. I chose `doctrine/cache` because it's simpler, well-tested and more stable.

So,
1. Are you sure you want me to rewrite it using `illuminate/cache`?
2. What cache driver support to implement next? SQLite3, Redis, Memcached or other?